packages/wrangler/src/__tests__/deploy.test.ts

```diff
@@ -9786,7 +9786,7 @@ addEventListener('fetch', event => {});`
 					)
 				)
 			).resolves.toMatchInlineSnapshot(`
-				"X [ERROR] Unexpected external import of \\"node:stream\\" and \\"node:timers/promises\\".
+				"X [ERROR] Unexpected external import of \\"node:events\\", \\"node:net\\", \\"node:stream\\", \\"node:timers/promises\\", and \\"node:tty\\".
```

Comment:
Unexpected change given the details in the changeset...
I assume that this release of unenv also injects these extra imports to support globals?

Reply:
Ah I see this in the PR description:

> One more https://github.com/unjs/unenv/pull/381 in unenv implements stdout, stderr and stdin with node:tty. It explains the added deps in tests.

Can you add that to the changeset?